apollo-router/src/plugins/telemetry/config_new/graphql/mod.rs

```diff
@@ -278,7 +303,7 @@ pub(crate) mod test {
                 2.0,
                 "graphql.field.name" = "users",
                 "graphql.field.type" = "User",
-                "graphql.type.name" = "Query"
+                "graphql.type.name" = "User"
```

Comment:
I still don't totally understand the different between `graphql.field.type` and `graphql.type.name`. For now, they're the same thing because the current implementation of the visitor doesn't share the parent's type. It looks like we have some code that should map types to one of `list`, `object`, `scalar`, etc. Is that meant to be used here?